actors/abi/epoch.go

```diff
@@ -0,0 +1,8 @@
+package abi
+
+// The duration of a chain epoch.
+// This is used for deriving epoch-denominated periods that are more naturally expressed in clock time.
+const EpochDurationSeconds = 25
```

Comment:
is this the variable we should use instead of [`BlockTimeSeconds` in `reward_state.go`](https://github.com/filecoin-project/specs-actors/blob/master/actors/builtin/reward/reward_state.go#L138) soon?

Reply:
I believe so, will let @anorth confirm

Reply:
Yes

Reply:
can we make some of these vars? that way we can change them live in the implementations for testing things.

Reply:
it would pain me, but in lieu of taking time to make them properly configurable, I'm on board.

Reply:
will let you guys make the change here in a different PR

Reply:
I tried, but the change must flow on to all the other constants that are derived from this value, which need to read the live-changed value rather than the initial one set here, and thus be functions. I opted not to force this on us so close to testnet-2.